hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapBasicTest.java

```diff
@@ -1076,8 +1077,29 @@ public void testGetEntryView() {
 
         assertEquals(key, view.getKey());
         assertEquals(value, view.getValue());
+        assertEquals(Long.MAX_VALUE, view.getMaxIdle());
```

Comment:
Looks like these two are the only two methods test the methods with `maxIdle` support. Shouldn't be more here?

Also, these are unrelated to this PR though:
- `testGetAll_whenMapEmpty` is missing `@Test` annotation
- many tests declare thrown exceptions while no invoked method is throwing checked exceptions

Reply:
Thanks @blazember. 
I fixed the missing annotation & the exceptions.
Regarding the tests, I added 1 more to handle the map config value (when user gives negative input), but these are the tests handling `maxIdle` from an entry-view perspective. We test entries with maxIdle in the `EvictionTest` for both OSS/EE & client.